doc/manual/packages.rst

```diff
@@ -717,7 +717,8 @@ This creates a commit in the ``~/.julia/v0.4/METADATA`` repo::
 This commit is only locally visible, however.  In order to make it visible to
 the world, you need to merge your local ``METADATA`` upstream into the official
 repo.  The :func:`Pkg.publish` command will fork the ``METADATA`` repository on
-GitHub, push your changes to your fork, and open a pull request::
```

Comment:
I see. I only did this once so far and assumed that it only does it once if ssh-agent is set up.  If there is a way to avoid it, then we should include it here.